staging/src/k8s.io/apiserver/pkg/audit/metrics.go

```diff
@@ -81,7 +81,7 @@ func HandlePluginError(plugin string, err error, impacted ...*auditinternal.Even
 	msg := fmt.Sprintf("Error in audit plugin '%s' affecting %d audit events: %v\nImpacted events:\n",
 		plugin, len(impacted), err)
 	for _, ev := range impacted {
-		msg = msg + EventString(ev) + "\n"
+		msg = msg + EventString(ev)
```

Comment:
Actually I want the EventStrings to be separated by a newline. This could be fixed either by moving the `\n` to before the EventString (i.e. `msg = msg + "\n" + EventString(ev)`) and removing the trailing newline from the initial msg, or perhaps more simply just change the log statement to `glog.Errorf`